testsuite/manualmode/src/test/java/org/jboss/as/test/manualmode/management/cli/ReloadRedirectTestCase.java

```diff
@@ -226,6 +226,7 @@ public void testReloadwithRedirect() throws Exception {
             promptFound = cliProc.pushLineAndWaitForResults("reload", "Accept certificate");
             assertTrue("No certificate prompt " + cliProc.getOutput(), promptFound);
         } finally {
+            cliProc.ctrlCAndWaitForClose();
             cliProc.destroyProcess();
```

Comment:
@pkremens , not sure you need to call destroyProcess once CtrlC has been sent. ctrlCAndWaitForClose() should be enough.